### PR TITLE
[loki-distributed] updating for loki release 2.2.1

### PR DESCRIPTION
**Pull request by nvanheuverzwijn**
Signed-off-by: Nicolas Vanheuverzwijn <nicolas.vanheu@gmail.com>

### CODE REVIEW

**Comment by sherifkayad** [comment]
@nvanheuverzwijn I guess this is a very good PR. Can you rebase to the latest changes introduced to the `main` branch and adjust this PR accordingly?

File: charts/loki-distributed/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
-appVersion: 2.2.0
-version: 0.31.1
+appVersion: 2.2.1
+version: 0.32.0
```
Comment (Whyeasy) [inline charts/loki-distributed/Chart.yaml:6]: It's a minor upgrade for Loki and no big changes. You can use a minor incremental for the Helm chart.

**Review by Whyeasy — CHANGES_REQUESTED** [review]

File: charts/loki-distributed/Chart.yaml (hunk 1 above)
Comment (nvanheuverzwijn) [reply charts/loki-distributed/Chart.yaml]: @Whyeasy Done

**Review by Whyeasy — APPROVED** [review]
LGTM!

**Pull request merged**